docs/content/en/docs/cluster-operator/vms-deploy.md

```diff
@@ -209,7 +209,70 @@ kube-system   nodelocaldns-fpfxj                 1/1     Running   0          8m
 
 We can see that the cluster on VMs is installed successful.
 
-## Delete the k8s cluster for VMs
+## Scale nodes on VMs
+
+With Kurator, you can declaratively add, remove, or replace multiple worker nodes on VMs.
+
+When performing scaling, you should avoid modifying the hostname in case the same VM has multiple names configured.
+
+You only need to declare the desired final worker node state on the target customMachine, and Kurator can complete the node scaling without any external intervention.
+
+You can make a copy of the custommachine.yaml file and edit it to reflect the desired scaling state.
+
+```console
+cp examples/infra/my-customcluster/cc-custommachine.yaml examples/infra/my-customcluster/scale.yaml
+vi examples/infra/my-customcluster/scale.yaml
```

Comment:
You can simply provide a yaml under example for up/down

Reply:
ok